.travis.yml

```diff
@@ -161,14 +185,13 @@ install:
     - test "$TRAVIS_OS_NAME" != "linux" || (scripts/install_cmake.sh)
     # Disable tests unless run on the release branch, on tags or with daily cron
     #- if [ "$TRAVIS_BRANCH" != release -a -z "$TRAVIS_TAG" -a "$TRAVIS_EVENT_TYPE" != cron ]; then SOLC_TESTS=Off; fi
-    - SOLC_TESTS=Off
```

Comment:
This will enable the tests.

Reply:
I know it is probably good to have this while this branch is in progress, but I think we should remove it again, or at least limit the tests to `--no-ipc` or just a single EVM version.

Reply:
Yes, we can disable them again, but I thought it'd be good to have to verify that everything works correctly for this PR.

Reply:
I'd leave them in until we agree that everything else is fine and only disable them again as a last step before merging.